### ISSUE TITLE
./easyrsa: 644: ./easyrsa: [[: not found

### ISSUE THREAD

**Issue by dpedu2** [OpenVPN/easy-rsa#164]
When I run the command:

```
 ./easyrsa --subject-alt-name='DNS:*.mydomain.com' sign-req server '*.mydomain.com'
```

I get the error:

```
./easyrsa: 644: ./easyrsa: [[: not found
```

Context:

```
$ ./easyrsa --subject-alt-name='DNS:*.mydomain.com' sign-req server '*.mydomain.com'


You are about to sign the following certificate.
Please check over the details shown below for accuracy. Note that this request
has not been cryptographically verified. Please be sure it came from a trusted
source or that you have verified the request checksum with the sender.

Request subject, to be signed as a server certificate for 3650 days:

subject=
    commonName                = *.mydomain.com


Type the word 'yes' to continue, or any other input to abort.
  Confirm request details: yes
./easyrsa: 644: ./easyrsa: [[: not found
Using configuration from ./openssl-1.0.cnf
Enter pass phrase for /home/ubuntu/easy-rsa3/EasyRSA-3.0.3/pki/private/ca.key:
Check that the request matches the signature
Signature ok
The Subject's Distinguished Name is as follows
commonName            :T61STRING:'*.mydomain.com'
Certificate is to be certified until Nov 27 23:05:59 2027 GMT (3650 days)
failed to update database
TXT_DB error number 2

Easy-RSA error:

signing failed (openssl output above may have more detail)
```

It looks like `easyrsa` uses sh, but `[[` is a bash builtin.

I believe the other error, regarding TXT_DB, is my fault (as I was trying to sign a cert matching one that already exists).

Edit: with a fresh copy of easyrsa, the issue persists (through I don't get the TXT_DB again).

**Comment by TinCanTech** [comment]
Duplicate of https://github.com/OpenVPN/easy-rsa/issues/137 and fixed 4ba461d

**Comment by dpedu2** [comment]
Oops, thanks!

**Comment by ecrist** [comment]
As @TinCanTech said, fixed in 4ba461d
